### ISSUE TITLE
Add strict option

### ISSUE THREAD

**Issue by Hito01** [pgilad/leasot#84]
Hi there,

I've tried to use the CLI with additionnal tags `PROD` and `STAGING` the problem is I have a comment in a PHP file :

```
// Products list
```

The reporter outputs this :

```
{
    "file": "path/file.php",
    "kind": "PROD",
    "line": 32,
    "text": "ucts list",
    "ref": ""
  },
```

I'd like to know if it would be possible to add a flag or something like that to strictly match the tags ? So if I summarize :

```
// Products list
```
Shouldn't be reported but :

```
// Prod : add this API key
```
should be

Cheers !

**Comment by pgilad** [comment]
Oh yeah that makes perfect sense, can you add a failing unit-test for this? I think this should be the default behavior since tag should not match loosely

**Comment by Hito01** [comment]
@pgilad Yeah. I've started to make to modifications in a fork but I don't know if you want that to be the default behavior or handle the behavior with a flag. Just let me know, I'll try to help after that.

Thanks !

**Comment by pgilad** [comment]
I think we should make the tags strict by default, and bump major version to avoid breaking changes

**Comment by pgilad** [comment]
Hey @Hito01 I added a PR to handle this issue, let me know what you think and if it works well for you:
https://github.com/pgilad/leasot/pull/98
